edxval/tests/test_api.py

```diff
@@ -2480,6 +2480,37 @@ def test_create_or_update_video_exceptions(self, video_id, file_format, provider
 
         self.assertEqual(transcript_exception.exception.message, exception_message)
 
+    @mock.patch.object(VideoTranscript, 'save')
+    def test_create_or_update_transcript_atomicity(self, mock_save):
+        """
+        Verify `create_or_update_video_transcript` api function atomicity.
+        """
+        file_data = ContentFile(constants.TRANSCRIPT_DATA['overwatch'])
+        file_name = None
+        file_format = utils.TranscriptFormat.SRT
```

Comment:
let's not create extra variable here. We will just pass value to method directly